src/dosbox.cpp

```diff
@@ -516,6 +516,9 @@ void DOSBOX_Init()
 	pstring->Set_help("Video memory in MB (1-8) or KB (256 to 8192). 'auto' uses the default per\n"
 	                  "video adapter ('auto' by default).");
 
+	pbool = secprop->Add_bool("force_vga_single_scan", when_idle, false);
+	pbool->Set_help("Always single-scan 200-line modes for VGA machine types");
+
```

Comment:
Technically, this should affect all sub-350 line modes, so maybe say that? E.g. all sub-350 line resolutions in the Quake demo such as 320x240 should be affected. The 400x300 custom mode would also be double scanned. Scorched Earth supports a lot of weird custom modes, that's a good test app.

Reply:
Tested and confirmed.